### DISCUSSION TITLE
v2.11.0-beta3

### DISCUSSION THREAD

**Discussion by fiftin** [semaphoreui/semaphore#2604] [Announcements]
## Bugfixes
* Terraform: check workspace support before use it (issue https://github.com/semaphoreui/semaphore/issues/2329)

## Features
* Renamed **Environment** to **Variable Group**
* Redesign Variable Group editing form

   https://github.com/user-attachments/assets/a3590da8-643c-4992-bfcd-bfca7d3917d5

* Added Destroy flag Terraform tasks

   ![image](https://github.com/user-attachments/assets/ee365456-24e1-4ae7-8606-2dc028924565)

<hr /><em>This discussion was created from the release <a href='https://github.com/semaphoreui/semaphore/releases/tag/v2.11.0-beta3'>v2.11.0-beta3</a>.</em>

**Comment by madozu** [comment]
Great to see the "-destoy" option 😃 ... works like a charm, at least for me. Thanks @fiftin 👏
